xo.py

```diff
@@ -0,0 +1,180 @@
+import random
+import string
+
+ALL_LETTERS = string.ascii_lowercase
+DEFAULT_BOARD_SIZE = 3
+
+
+def start_game(board_size=DEFAULT_BOARD_SIZE):
+
+    players = gather_players_names()
+
+    while True:
+        # Randomize play order.
+        random.shuffle(players)
+
+        print('First player to start is: ' + players[0])
```

Comment:
Better to use `f-strings` when possible.

Reply:
Done